go.sum

```diff
@@ -56,6 +56,8 @@ github.com/Azure/go-autorest/tracing v0.5.0/go.mod h1:r/s2XiOKccPW3HrqB+W0TQzfbt
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/DATA-DOG/go-sqlmock v1.3.3/go.mod h1:f/Ixk793poVmq4qj/V1dPUg2JEAKC73Q5eFN3EC/SaM=
+github.com/INFURA/go-ethlibs v0.0.0-20230210163729-fc6ca4235802 h1:GGU/jUNUUkGbnnqi7Eoq8L9qEDG+GbW5Z/sCuYFPK9U=
```

Comment:
Which variables are we using from go-ethlibs?

Reply:
We're importing this library: https://pkg.go.dev/github.com/INFURA/go-ethlibs/jsonrpc#Response
It's very nice for dealing with the type of JSON RPCs that include arrays in the "Param" field by default, which is what go-quai and geth use extensively.

Incidentally, there are some improvements I'd like to see/implement and I've had an issue up in the proxy to address them. https://github.com/dominant-strategies/go-quai-stratum/issues/7